### ISSUE TITLE
Two cubes appear in the url bar when Beta and Stable are installed

### ISSUE THREAD

**Issue by olizilla** [ipfs-shipyard/ipfs-companion#733]
Not a huge issue, but i figured i'd log it, i was checking what our error messages and got two ipfs cubes in my url bar in firefox

https://ipfs.io/ipns/x.foo

<img width="964" alt="Screenshot 2019-07-12 at 16 36 29" src="https://user-images.githubusercontent.com/58871/61140528-85178580-a4c3-11e9-8fbf-02cd809068ea.png">

**Comment by olizilla** [comment]
nope! My bad. @alanshaw just pointed out that I have 2 companions installed.

**Comment by lidel** [comment]
Having both Beta and Stable versions is indeed very confusing, and I'd like to repurpose this issue and see if we can do anything to detect that and provide user with better UX.

Initial idea:

- detect we are running a Beta version, and then try to ping Stable version via [`onConnectExternal`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onConnectExternal) & [`onMessageExternal`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessageExternal)
- if Stable version is present, Beta version should not start, but open a static page in a new tab informing user that they need to uninstall Stable if they want to use Beta. 

@olizilla  Thoughts?
